Makefile

```diff
@@ -15,27 +17,34 @@ metastore-shell = cd $(metastore) && nix-shell https://holochain.love --pure --c
 .PHONY: squad-games-web
 squad-games-web: build/bootstrap $(curation-market-js)/curation-config.json
 squad-games-web: $(js-client-contracts) build/devnet metastore
-	cd $(squad-games-web) && npm run load_dev_defs
 	cd $(squad-games-web) && npm run start
 
 
-.PHONY: app-spec-web
-app-spec-web: build/bootstrap $(curation-market-js)/curation-config.json
-app-spec-web: $(js-client-contracts)
-	cd $(app-spec-web) && npm run start
+.PHONY: app-spec-roshambo
+app-spec-roshambo: build/bootstrap $(curation-market-js)/curation-config.json
+app-spec-roshambo: $(js-client-contracts)
+	cd $(app-spec-roshambo) && npm run start
+
+
+.PHONY: squad-chess
+squad-chess: build/bootstrap
+# TODO make this depend on metastore like things depend on build/devnet
+	cd $(squad-chess) && node load_defs.js
+	cd $(squad-chess) && npm run start
```

Comment:
I think the command is actually just `npm start` unless you changed that.

Reply:
looks like `npm test`, `npm start`, `npm restart`, and `npm stop` are all aliases for their `npm run ...` counterparts.